django/views/i18n.py

```diff
@@ -318,3 +331,118 @@ def json_catalog(request, domain='djangojs', packages=None):
         'plural': plural,
     }
     return http.JsonResponse(data)
+
+
+class JavascriptCatalog(View):
+    """
+    Returns the selected language catalog as a javascript library.
+
+    Receives the list of packages to check for translations in the
+    kwargs `packages` key either from an infodict or as a +-delimited
```

Comment:
I'm not sure what "infodict" means, I see it's copied from the existing docstring. Any idea?